app/src/gui/components/record/formButton.tsx

```diff
@@ -152,9 +146,41 @@ export default function FormButtonGroup({
   views,
   ui_specification,
   layout,
-  showPublishButton = true,
+  publishButtonBehaviour,
+  showPublishButton,
 }: FormButtonGroupProps) {
   const [tooltipOpen, setTooltipOpen] = useState<boolean>(false);
+  const [openCancelDialog, setOpenCancelDialog] = useState(false);
+
+  /**
+   * Tooltip content explaining the publish action
+   */
+  const tooltipContent = (
+    <div>
+      <Box sx={{fontWeight: 'bold', marginBottom: 1}}>
+        What does finish mean?
+      </Box>
+      <Typography variant="body1">
+        Your response is being saved automatically as a draft. When you click
+        finish, your response will be uploaded once your device has an internet
+        connection.
+      </Typography>
+      {publishButtonBehaviour === 'noErrors' && (
+        <Typography variant="body1">
+          You cannot finish this record because there are errors in the form.
+        </Typography>
+      )}
```

Comment:
shouldn't this only appear if the behaviour is noErrors and there are errors - like the 'showPublishButton' is false?

Reply:
Yes, and the one below.